static/lib/style.css

```diff
@@ -0,0 +1,514 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+* {
+    font-family: 'Poppins', sans-serif;
+    padding: 0;
+    margin: 0;
+    text-decoration: none;
+    box-sizing: border-box;
+    list-style: none;
+}
+
+:root {
+    --color-main: #142160;
+    --bg: #fbfefd;
+    --bg2: #dce5ff6c;
+    --main-accent: #e9eefd;
+    --main-text: #4b5976;
+    --shadow: rgba(17, 17, 26, 0.1) 0px 0px 16px;
+
+}
+
```

Comment:
Same here; please check the rest of the file as well; I see some inconsistent extra empty line / no new line between styles (unless you intended)